nsqadmin/static/js/views/app.js

```diff
@@ -136,6 +136,10 @@ var AppView = BaseView.extend({
     },
 
     onLinkClick: function(e) {
+        if (e.ctrlKey || e.metaKey) {
+          // allow ctrl+click to open in a new tab
+          return true
```

Comment:
I'm not sure i follow. Are you implying an empty `return`?

`return false` should stop the browser navigation to the `href`. That would mean clicking does nothing. `return true` (from experimentation) allows the browser to navigate and do a full page load on the `href` target (though because ctrl or meta, i **expect** the browser to be doing that in a new tab)

Why would i want to `stopPropagation`?